src/mumble/Settings.cpp

```diff
@@ -401,7 +401,16 @@ Settings::Settings() {
 	bJackStartServer  = false;
 	bJackAutoConnect  = true;
 
-	echoOption = EchoCancelOptionID::DISABLED;
+#ifdef Q_OS_MAC
+	// On macOS Speex can't be used, so we default to Apple's custom echo cancellation mode
+	// Note that this only seems to work with when using the builtin microphone and the builtin speakers. It
```

Comment:
`// Note that this only seems to work when using the built-in microphone and the built-in speakers. It`